imperial.js

```diff
@@ -244,12 +246,12 @@ class Imperial {
 
   nations() {
     let nations = {};
-    ["AH", "IT", "FR", "GB", "GE", "RU"].map((nation) => {
+    for (const nation of Nation) {
```

Comment:
Not that this is a good idea, but is it conceivably possible for us to give `FrozenSet` a function like `map` so that we can call `Nation.map((nation) => ...)`? Or maybe make another constant, which is `Nations` (plural) and _that_ refers to the iterable array of them?

Reply:
I can definitely make a map function. It'll look something like this:

```es6
function map (fn) { [...this].map(fn) }
```